nx migrate is missing zone.js

### Current Behavior

running `nx migrate latest` update all of the `@angular/*` packages, but miss out `zone.js` which makes `npm i` fails

### Expected Behavior

`nx migrate latest` should update `zone.js` to latest as well

### GitHub Repo

_No response_

### Steps to Reproduce

1. generate new workspace with angular 17
2. update to angular 18 using `nx migrate latest`

### Nx Report

```shell
Node           : 20.14.0
OS             : win32-x64
Native Target  : x86_64-windows
npm            : 10.8.3

nx                 : 19.8.0
@nx/js             : 19.8.0
@nx/jest           : 19.8.0
@nx/linter         : 19.8.0
@nx/eslint         : 19.8.0
@nx/workspace      : 19.8.0
@nx/angular        : 19.8.0
@nx/cypress        : 19.8.0
@nx/devkit         : 19.8.0
@nx/eslint-plugin  : 19.8.0
@nrwl/tao          : 19.8.0
@nx/web            : 19.8.0
@nx/webpack        : 19.8.0
typescript         : 5.5.4
```


### Failure Logs

```shell
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @angular/core@18.2.5
npm error Found: zone.js@0.14.6
npm error node_modules/zone.js
npm error   zone.js@"0.14.6" from the root project
npm error
npm error Could not resolve dependency:
npm error peer zone.js@"~0.14.10" from @angular/core@18.2.5
npm error node_modules/@angular/core
npm error   @angular/core@"18.2.5" from the root project
npm error   peer @angular/core@"18.2.5" from @angular/animations@18.2.5
npm error   node_modules/@angular/animations
npm error     @angular/animations@"18.2.5" from the root project
npm error     1 more (@angular/platform-browser)
npm error   7 more (@angular/common, @angular/compiler, @angular/forms, ...)
npm error
npm error Conflicting peer dependency: zone.js@0.14.10
npm error node_modules/zone.js
npm error   peer zone.js@"~0.14.10" from @angular/core@18.2.5
npm error   node_modules/@angular/core
npm error     @angular/core@"18.2.5" from the root project
npm error     peer @angular/core@"18.2.5" from @angular/animations@18.2.5
npm error     node_modules/@angular/animations
npm error       @angular/animations@"18.2.5" from the root project
npm error       1 more (@angular/platform-browser)
npm error     7 more (@angular/common, @angular/compiler, @angular/forms, ...)
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```


### Package Manager Version

_No response_

### Operating System

- [ ] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

workaround is `npm i zone.js@0.14.10` where 0.14.10 is the version in the error message

--- a/packages/angular/migrations.json
+++ b/packages/angular/migrations.json
@@ -1127,6 +1127,10 @@
         "ng-packagr": {
           "version": "~18.2.0",
           "alwaysAddToPackageJson": false
+        },
+        "zone.js": {
+          "version": "~0.14.10",
+          "alwaysAddToPackageJson": false
         }
       }
     },

